webrtc.html

```diff
@@ -509,33 +509,42 @@
 
         <ol>
           <li>
-            <p>Append an object representing the current call being handled
-            (i.e. function name and corresponding arguments) to the
-            <code>operations</code> array.</p>
+            <p>If <var>connection</var>'s <a href=
+              "#dom-peerconnection-signaling-state"><code>RTCPeerConnection</code>
+              signalingState</a> is <code>closed</code>, throw an
+              <code>InvalidStateError</code> exception and abort these steps.</p>
           </li>
 
           <li>
-            <p>If the length of the <code>operations</code> array is exactly 1,
-            execute the function from the front of the queue
-            asynchronously.</p>
+            <p>Append an <code>operation</code> to
+            the <var>operations</var> queue, where
+            an <code>operation</code> represents the asynchronous
+            execution of the called function that will fulfill the
+            promise returned by the function.</p>.
```

Comment:
The operation is the asynchronous execution that will fulfill the promise.  The code that sticks the operation in the queue is run synchronously, but surely the fulfillment of the promise isn't.